release_dashboard/lib/enums/repositories.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A toggle enum to switch between engine and framework.
+enum Repositories {
```

Comment:
Let's rebase once the enums PR has landed, then re-request my review (this mostly lgtm)

Reply:
Done, please take another look.